src/main/java/io/github/dsheirer/module/decode/config/WithCallTimeout.java

```diff
@@ -0,0 +1,12 @@
+package io.github.dsheirer.module.decode.config;
+
+/**
+ * Trait of decoder configurations that support providing information about call timeout.
+ */
+public interface WithCallTimeout
+{
+    /**
+     * @return call timeout in seconds
+     */
+    int getCallTimeout();
```

Comment:
or `getCallTimeoutSeconds` to make it explicit?

Reply:
The way you have it coded is good.  The javadoc clarifies it.